### PR TITLE
[1.3.1 patch] Update CHANGELOG with the new issue

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Update CHANGELOG with the issue #3629

### CODE REVIEW

File: src/OpenTelemetry.Api/CHANGELOG.md [hunk 1]
```diff
@@ -177,8 +187,9 @@ Released 2021-Jan-29
 * Relax System.Diagnostics.DiagnosticSource version requirement to allow
   versions >=5.0. Previously only versions up to 6.0 (excluding 6.0) was
   allowed.
-* Metrics removed as it is not part 1.0.0 release. See issue [#1501](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1655)
-  for details on Metric release plans.
+* Metrics removed as it is not part 1.0.0 release. See issue
+  [#1501](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1655) for
```
Comment (reyang) [inline src/OpenTelemetry.Api/CHANGELOG.md:191]: The text and link are actually pointing to two different locations?

File: src/OpenTelemetry.Api/CHANGELOG.md [hunk 2]
```diff
@@ -100,10 +108,12 @@ Released 2021-Aug-24
 
 Released 2021-Jul-23
 
-* Add Metrics support.([#2174](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2174))
+* Add Metrics
+  support.([#2174](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2174))
```
Comment (reyang) [inline src/OpenTelemetry.Api/CHANGELOG.md:114]: It seems this missed a space:

```suggestion
  support. ([#2174](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2174))
```

**Review by reyang — APPROVED** [review]
LGTM.

**Comment by utpilla** [comment]
Closing this in favor of #3631

**Pull request closed without merging**